exercises.md

```diff
@@ -10,7 +10,7 @@ First we are going to run **hello-world** container prepared by Docker Team. Ope
 docker run hello-world
 ```
 
-> More information about above comand: [https://docs.docker.com/engine/getstarted/step\_two/](https://docs.docker.com/engine/getstarted/step_two/)
+> More information about above comand [click here](https://docs.docker.com/get-started/part2/).
```

Comment:
I would use this link: https://docs.docker.com/engine/reference/commandline/run/